初始化 ice design pro 项目后，运行时报错（Babel 依赖问题？）

<!--
## 基本规范

- 避免直接给一个错误场景，尽量做一些简单的跟踪分析，然后描述清楚问题
- 组件问题请打开下面的链接书写重现 demo 并描述清楚重现路径
  - @alifd/next(1.x): https://codepen.io/sobear/pen/NeJmOG?editors=0011
  - @icedesign/base(0.x): https://codepen.io/sobear/pen/zybQYj?editors=0011
- `@alifd/next` 的组件问题请在[这里](https://fusion.design/feedback)提交 issue，其他组件问题请在本仓库提交 issue
- 构建问题请提供 `package.json` 以及 `.webpackrc.js` 的内容
- 工具问题（Iceworks/ice-scripts/ice-devtools）请提供实际安装的版本号、系统环境、Node.js 版本
-->

## 问题描述

类似问题：<https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues/62> <https://github.com/dvajs/dva/issues/1980>
问题来源：<https://github.com/babel/babel/pull/8266>

## 错误信息

```
Failed to compile
./node_modules/react-redux/es/components/connectAdvanced.js
Module not found: Can't resolve '@babel/runtime/helpers/esm/assertThisInitialized' in '/Users/songkeys/Github/uCourse-mp/node_modules/react-redux/es/components'
```

--- a/tools/ice-scripts/CHANGELOG.md
+++ b/tools/ice-scripts/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.9.1
+
+- [fix] 修复 @babel/runtime alias 路径错误问题
+
 ## 1.9.0
 
 - [feat] 支持 init 项目

--- a/tools/ice-scripts/lib/config/getResolveAlias.js
+++ b/tools/ice-scripts/lib/config/getResolveAlias.js
@@ -37,7 +37,7 @@ module.exports = function getResolveAlias(buildConfig) {
   }
 
   // 项目不需要单独依赖 @babel/runtime
-  alias['@babel/runtime'] = path.resolve(__dirname, '../../node_modules/@babel/runtime');
+  alias['@babel/runtime'] = path.resolve(require.resolve('@babel/runtime/package.json'), '../');;
 
   return alias;
 };

--- a/tools/ice-scripts/lib/config/getResolveAlias.js
+++ b/tools/ice-scripts/lib/config/getResolveAlias.js
@@ -37,7 +37,7 @@ module.exports = function getResolveAlias(buildConfig) {
   }
 
   // 项目不需要单独依赖 @babel/runtime
-  alias['@babel/runtime'] = path.resolve(require.resolve('@babel/runtime/package.json'), '../');;
+  alias['@babel/runtime'] = path.resolve(require.resolve('@babel/runtime/package.json'), '../');
 
   return alias;
 };

--- a/tools/ice-scripts/package.json
+++ b/tools/ice-scripts/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ice-scripts",
-  "version": "1.9.0",
+  "version": "1.9.1",
   "description": "ICE SDK",
   "main": "index.js",
   "bin": {

